### PR TITLE
Cs registration del

### PR DESCRIPTION
**Pull request by Neeraj-Nec**
delete csource code

### CODE REVIEW

File: broker/restapisrv.go [hunk 1]
```diff
@@ -62,35 +60,21 @@ func (apisrv *RestApiSrv) Start(cfg *Config, broker *ThinBroker) {
...
-		//create and update
+		rest.Post("/ngsi-ld/v1/notifyContext/", broker.NotifyLdContext),
 		rest.Post("/ngsi-ld/v1/entities/", broker.LDCreateEntity),
+		rest.Get("/ngsi-ld/v1/entities/#eid", apisrv.LDGetEntity),
+		rest.Get("/ngsi-ld/v1/entities", apisrv.GetQueryParamsEntities),
 		rest.Post("/ngsi-ld/v1/entities/#eid/attrs", broker.LDAppendEntityAttributes),
 		rest.Patch("/ngsi-ld/v1/entities/#eid/attrs", broker.LDUpdateEntityAttributes),
 		rest.Patch("/ngsi-ld/v1/entities/#eid/attrs/#attr", broker.LDUpdateEntityByAttribute),
-
-		//query
```
Comment (naveennec) [inline broker/restapisrv.go:72]: this should not be deleted, its comments which tells about the below API

File: broker/restapisrv.go [hunk 2]
```diff
@@ -62,35 +60,21 @@ func (apisrv *RestApiSrv) Start(cfg *Config, broker *ThinBroker) {
...
-		rest.Get("/ngsi-ld/v1/entities/#eid", apisrv.LDGetEntity),
-		rest.Get("/ngsi-ld/v1/entities", apisrv.GetQueryParamsEntities),
-
-		//delete
```
Comment (naveennec) [inline broker/restapisrv.go:76]: this should not be deleted, its comments which tells about the below API

File: broker/restapisrv.go [hunk 3]
```diff
@@ -62,35 +60,21 @@ func (apisrv *RestApiSrv) Start(cfg *Config, broker *ThinBroker) {
...
 		rest.Delete("/ngsi-ld/v1/entities/#eid", apisrv.DeleteLDEntity),
 		rest.Delete("/ngsi-ld/v1/entities/#eid/attrs/#attr", broker.LDDeleteEntityAttribute),
 
-		//subscription
```
Comment (naveennec) [inline broker/restapisrv.go:80]: this should not be deleted, its comments which tells about the below API

File: broker/restapisrv.go [hunk 4]
```diff
@@ -62,35 +60,21 @@ func (apisrv *RestApiSrv) Start(cfg *Config, broker *ThinBroker) {
...
 		rest.Post("/ngsi-ld/v1/subscriptions/", broker.LDCreateSubscription),
 		rest.Get("/ngsi-ld/v1/subscriptions/", broker.GetLDSubscriptions),
 		rest.Get("/ngsi-ld/v1/subscriptions/#sid", apisrv.GetLDSubscription),
 		rest.Patch("/ngsi-ld/v1/subscriptions/#sid", broker.UpdateLDSubscription),
 		rest.Delete("/ngsi-ld/v1/subscriptions/#sid", apisrv.DeleteLDSubscription),
-
-		//notify
```
Comment (naveennec) [inline broker/restapisrv.go:87]: this should not be deleted, its comments which tells about the below API

**Review by naveennec — CHANGES_REQUESTED** [review]

File: broker/restapisrv.go (hunk 1 above)
Comment (Neeraj-Nec) [reply broker/restapisrv.go:72]: Done in other pull request

Comment (Neeraj-Nec) [reply broker/restapisrv.go:72]: Dene in other pull request

Comment (Neeraj-Nec) [reply broker/restapisrv.go:72]: Done in other pull request

File: broker/restapisrv.go (hunk 2 above)
Comment (Neeraj-Nec) [reply broker/restapisrv.go:76]: Done in other pull request

File: broker/restapisrv.go (hunk 3 above)
Comment (Neeraj-Nec) [reply broker/restapisrv.go:80]: Done in other pull request

File: broker/restapisrv.go (hunk 4 above)
Comment (Neeraj-Nec) [reply broker/restapisrv.go:87]: Done in other pull request

**Pull request closed without merging**